unlock-app/src/components/interface/locks/Settings/forms/CreditCardForm.tsx

```diff
@@ -117,14 +117,15 @@ const ConnectStripe = ({
 
   const grantKeyGrantorRoleMutation = useMutation(async (): Promise<any> => {
     const walletService = await getWalletService(network)
-    return walletService.addKeyGranter({
+    return await walletService.addKeyGranter({
```

Comment:
does await change behavior here? It shouldn't 🤔

Reply:
no, does not change it